datetime/src/test/java/com/fasterxml/jackson/datatype/jsr310/deser/key/ZonedDateTimeKeyDeserializerTest.java

```diff
@@ -0,0 +1,70 @@
+package com.fasterxml.jackson.datatype.jsr310.deser.key;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class ZonedDateTimeKeyDeserializerTest {
+
+    private static ObjectMapper objectMapper;
+    private final TypeReference<Map<ZonedDateTime, String>> MAP_TYPE_REF = new TypeReference<Map<ZonedDateTime, String>>() {
+		};
+
+    @BeforeClass
+    public static void beforeClass() {
+        objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+    }
+
+    @Test
+    public void Instant_style_can_be_deserialized() throws JsonProcessingException {
```

Comment:
could you change this to `throws Exception` -- just so it's easier to merge into `master` branch (Jackson 3.0) where `JsonProcessingException` is replaced by `StreamReadException`.
(and same for other instances)

Reply:
Done